### DISCUSSION TITLE
2020 买房记录 - Xheldon 的生活

### DISCUSSION THREAD

**Discussion by Xheldon** [Xheldon/x_blog#120] [Announcements]
https://www.xheldon.com/life/buy-house-in-beijing.html 

 “总体还算顺利，没有太多的坑”

**Comment by northernsong** [comment]
我居然看完了！ 感觉分享

**Comment by Xheldon** [comment]
> @northernsong
> 我居然看完了！ 感觉分享

感谢喜欢😆
